core/tests/data/dummy_assets_yaml/assets/images/sample_Img.svg

```diff
@@ -0,0 +1 @@
+<svg width="100" height="100"><circle cx="50" cy="50" r="40" stroke="green" stroke-width="4" fill="yellow" /></svg>
```

Comment:
Please add a newline at EOF, for this and the next file. You can see that these are missing if you look at the "Files Changed" tab on GitHub.